### PR TITLE
Clarify the use of `/usr/local`

### PR DESCRIPTION
**Pull request by jimmykarily**

### CODE REVIEW

File: content/en/docs/Architecture/immutable.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ A running Linux-based OS system will have the following paths:
 ```
 
 `/usr/local` will contain all the persistent data which will be carried over in-between upgrades, unlike the changes made to `/etc` which will be discarded.
+NOTE: The persistent data live in directories that are bind mounted to `/usr/local` (check the [bind_mounts configuration option]({{< relref "../reference/configuration" >}})). Creating a directory in `/usr/local` manually (e.g. using a command in some `stage`) won't work.
```
Comment (mauromorales) [inline content/en/docs/Architecture/immutable.md:24]: I think an alert would be better for this ... other than that lgtm

Comment (jimmykarily) [reply content/en/docs/Architecture/immutable.md]: Good point! I keep forgetting about "alert". Changed.

**Review by mauromorales — APPROVED** [review]
lgtm

File: content/en/docs/Architecture/immutable.md [hunk 2]
```diff
@@ -22,6 +22,10 @@ A running Linux-based OS system will have the following paths:
 
 `/usr/local` will contain all the persistent data which will be carried over in-between upgrades, unlike the changes made to `/etc` which will be discarded.
 
+{{% alert color="info" %}}
+The persistent data live in directories that are bind mounted to `/usr/local` (check the [bind_mounts configuration option]({{< relref "../reference/configuration" >}})). Creating a directory in `/usr/local` manually (e.g. using a command in some `stage`) won't work.
```
Comment (mudler) [inline content/en/docs/Architecture/immutable.md:26]:
```suggestion
The persistent data live in directories that are bind mounted to `/usr/local/.state` (check the [bind_mounts configuration option]({{< relref "../reference/configuration" >}})).  `/usr/local` is mounted during boot with the first  partition found labeled `COS_PERSISTENT`. 
```

File: content/en/docs/Architecture/immutable.md [hunk 3]
```diff
@@ -22,6 +22,10 @@ A running Linux-based OS system will have the following paths:
 
 `/usr/local` will contain all the persistent data which will be carried over in-between upgrades, unlike the changes made to `/etc` which will be discarded.
 
+{{% alert color="info" %}}
+The persistent data live in directories that are bind mounted to `/usr/local/.state` (check the [bind_mounts configuration option]({{< relref "../reference/configuration" >}})).  `/usr/local` is mounted during boot with the first  partition found labeled `COS_PERSISTENT`.
```
Comment (jimmykarily) [inline content/en/docs/Architecture/immutable.md:26]:
```suggestion
The persistent data live in directories that are bind mounted to `/usr/local/.state` (check the [bind_mounts configuration option]({{< relref "../reference/configuration" >}})).  `/usr/local` is mounted during boot with the first partition found labeled `COS_PERSISTENT`. 
```

**Pull request merged**